packages/web-frontend/cypress/support/utils.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import cheerio from 'cheerio';
+import { html as beautifyHtml } from 'js-beautify';
+
+export const equalStringsI = (a, b) => a.toLowerCase().localeCompare(b.toLowerCase()) === 0;
+
+/**
+ * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions
+ */
+export const escapeRegex = string => {
+  // $& means the whole matched string
+  return string.replace(/[.*+\-?^${}()|[\]\\]/g, '\\$&');
+};
+
+export const stripHtmlAttributes = (html, attrs) => {
+  const $ = cheerio.load(html);
+  attrs.forEach(attr => {
+    $('*').removeAttr(attr);
+  });
+
+  return $('body').html();
+};
+
+/**
+ * Taken from @cypress/snapshot
+ *
+ * @see https://github.com/cypress-io/snapshot/blob/master/src/utils.js
+ */
+export const serializeReactToHTML = jQueryEl => {
+  const html = jQueryEl[0].outerHTML;
+  const stripped = stripHtmlAttributes(html, ['clip-path', 'id', 'data-reactid', 'data-testid']);
```

Comment:
The attributes could be pulled out into a named const for easy reference

Reply:
Good point, I will do that!